src/image.rs

```diff
@@ -466,18 +466,18 @@ number_enum! {
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
 pub enum ImageOrder {
     /// Least significant byte first
-    LSBFirst,
+    LeastSignificantByteFirst,
     /// Most significant byte first
-    MSBFirst,
+    MostSignificantByteFirst,
```

Comment:
Any reason not to use `Lsb` / `Msb`? I think they are well-known acronyms.

Reply:
Okay, I pushed another commit that does this change.